src/shared/api/lib.ts

```diff
@@ -0,0 +1,6 @@
+export const queryStringify = (data: Record<string, unknown>) => {
+  const keys = Object.keys(data);
+  return keys.reduce((result, key, index) => {
+    return `${result}${key}=${data[key]}${index < keys.length - 1 ? '&' : ''}`;
+  }, '?');
```

Comment:
```suggestion
  return keys.reduce((acc, key, index) => `${acc}${acc === '?' ? '' : '&'}${key}=${data[key]}`, '?');
```
Шило на мыло: можно не обращать внимания. Просто не смог пройти мимо:
- использования return сразу после =>
- result вместо acc
- добавления & в конец а не в начало